apps/artemis-web/pages/api/auth/change-password.ts

```diff
@@ -14,11 +14,18 @@ const handler = nc()
   .put(async (req: NextApiRequestExtended, res: NextApiResponseExtended) => {
     try {
       const { old_password, new_password } = req.body;
+      const regex = /^(?=.*[A-Z])(?=.*[!@#$&*])(?=.*[0-9])(?=.*[a-z]).{8,}$/g;
```

Comment:
this is best to be in a single place (e.g., utils) and not replicated across components